content/docs/user-guide/experiment-management/experiments-overview.md

```diff
@@ -49,29 +45,35 @@ this:
 [compare]: /doc/user-guide/experiment-management/comparing-experiments
 [persistent]: /doc/user-guide/experiment-management/persisting-experiments
 
-## Initialize DVC Experiments on any project
-
-To use DVC Experiments you need a <abbr>DVC project</abbr> with a minimal
-structure and configuration. To avoid having to bootstrap DVC manually, the
-`dvc exp init` command lets you quickly onboard an existing project to the DVC
-Experiments workflow.
+## Save experiments
 
-It will create a simple `dvc.yaml` metafile, which codifies your planned
-experiments. This includes the locations for expected <abbr>dependencies</abbr>
-(data, parameters, source code) and <abbr>outputs</abbr> (ML models,
-<abbr>metrics</abbr>, etc.). These assume [sane defaults] but can be customized
-with the options of `dvc exp init`.
+DVC is not opinionated about your experiments workflow. To save an experiment,
+you may choose any of these workflows:
 
-💡 We recommend adding the `-i` flag to use its [interactive mode]. This will
-ask you how to run the experiments, and guide you through customizing the
-aforementioned locations (optional).
+- If you have a DVC [pipeline], use `dvc exp run` to both [run] your code
+  pipeline and save experiment results. `dvc exp run` also enables other
+  features like queuing many experiments at once.
+- If you do not have a DVC pipeline, you can add [DVCLive] to your Python code
```

Comment:
should this option come first? (more natural I guess)

Reply:
🤷 I debated the order here. I'm happy to move it up.

Reply:
Moved it up